man/timesyncd.conf.xml

```diff
@@ -91,6 +91,12 @@
         <varname>PollIntervalMaxSec=</varname> defaults to 2048 seconds.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ConnectionRetrySec</varname></term>
```

Comment:
please suffix with "=" like we usually do for configuration variables that expect a value. i.e.

`<varname>ConnectionRetrySec</varname>` → `ConnectionRetrySec=</varname>`